crates/gitbutler-branch-actions/tests/fixtures/for-workspace-migration.sh

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env bash
+set -eu -o pipefail
+CLI=${1:?The first argument is the GitButler CLI}
+
+git init remote
+(cd remote
+  echo first > file
+  git add . && git commit -m "init"
+)
+
+export GITBUTLER_CLI_DATA_DIR=../user/gitbutler/app-data
+git clone remote workspace-migration
+(cd workspace-migration
+  $CLI project add --switch-to-workspace "$(git rev-parse --symbolic-full-name @{u})"
+  $CLI branch create virtual
+  # Start the test on the old integration branch.
+  git checkout -b gitbutler/integration
+)
```

Comment:
@Byron I'm adding this file to follow the pattern set by other tests, but I'm a bit unsure how to think about the output in `generated-do-not-edit`. The associated test fails if run twice without removing the output. Is there something I should do in either this file, or in the rust code, such that I can run the tests repeatedly?

Reply:
I think @krlvi fell for this as well, as the tests are designated to be read-only. Many of the even read-only sounding operations actually write, which destroys the fixture.

If these are executed through Rust, I think @krlvi added ways to get a writable fixture, i.e. one that is created into a temp-dir.
Does this help? (I didn't look at the code here specifically)